microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -420,9 +420,9 @@ Microsoft Edge collects diagnostic data about how you use the new tab page, such
 
```

Comment:
@vwehren - fix the instance of "Office 365" in the "New tab page" parent section above (line 403).
Live article section:
https://learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/#new-tab-page